spec/features/admin_can_view_items_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+  create_admin
+  feature 'admin' do
+    it 'can view list of all items' do
+      create_rental_items
+
+      visit root_path
+
+      click_link "Login"
+
+      fill_in "Username", with: 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.